chain/client/src/stateless_validation/chunk_validator/orphan_witness_handling.rs

```diff
@@ -21,11 +20,6 @@ use std::ops::Range;
 /// have the previous block available (the chain head), so it wouldn't be an orphan.
 pub const ALLOWED_ORPHAN_WITNESS_DISTANCE_FROM_HEAD: Range<BlockHeight> = 2..6;
 
-/// We keep only orphan witnesses which are smaller than this size.
-/// This limits the maximum memory usage of OrphanStateWitnessPool.
-/// TODO(#10259) - consider merging this limit with the non-orphan witness size limit.
```

Comment:
Let's preserve the TODO - once normal witness size limit is implemented we should ensure that both normal and orphan limits are the same.

Reply:
My bad, moved to the struct Config now, thanks for noticing this.